### PR TITLE
Migrate setup scripts to v1

### LINKED ISSUE #1244: Fix installation docu and scripts
At least "plain" and "react" are not working, also check others.

For plain, remove any unnecessary dependency, e.g. webpack, from script and manual steps in docu, stick to live-server!

### LINKED ISSUE #1312: Migrate setup scripts to v1 and fundamental-styles
Migrate setup scripts to v1 and fundamental-styles.
Create a set of assets that are used for setup scripts to avoid search replace in those downloaded files. 
Update inline scripts in the setup guide.

### PR DESCRIPTION
**Pull request by maxmarkus**
Fixes #1312
Possibly fixes #1244

For testing follow these steps: 
- Go to _./scripts/setup/_
- Replace inside _setup/*.sh_:   `https://raw.githubusercontent.com/SAP/luigi/master/scripts/setup/assets` with `http://localhost:8005`
- In _./scripts/setup/assets_ run `ws --port 8005`

### CODE REVIEW

File: scripts/setup/react.sh [hunk 1]
```diff
@@ -15,35 +15,46 @@ npx create-react-app $folder && cd $folder
...
 mv public/index.html public/react.html
 curl https://raw.githubusercontent.com/SAP/luigi/master/core/examples/luigi-example-react/public/index.html > public/index.html
-curl https://raw.githubusercontent.com/SAP/luigi/master/core/examples/luigi-example-react/public/logo.png > public/logo.png
 curl https://raw.githubusercontent.com/SAP/luigi/master/core/examples/luigi-example-react/public/sampleapp.html > public/sampleapp.html
-curl https://raw.githubusercontent.com/SAP/luigi/master/core/examples/luigi-example-react/public/luigi-config.js > public/luigi-config.js
+curl https://raw.githubusercontent.com/SAP/luigi/master/core/examples/luigi-example-react/public/luigi-config.js > src/luigi-config/luigi-config.es6.js
+
 
 # add index.js
 curl https://raw.githubusercontent.com/SAP/luigi/master/core/examples/luigi-example-react/src/index.js > src/index.js
```
Comment (ndricimrr) [inline scripts/setup/react.sh:49]: The react script updated doesn't seem to work for me 🤔 . The reason I think is we install **luigi-project/client** and then fetch in this curl an **index.js** which contains **kyma-project/luigi** instead.  Let me know if this is the case for you as well.

File: scripts/setup/react.sh [hunk 2]
```diff
@@ -15,35 +15,46 @@ npx create-react-app $folder && cd $folder
...
 curl https://raw.githubusercontent.com/SAP/luigi/master/core/examples/luigi-example-react/src/index.css > src/index.css
 
-#add views
+# add views
 mkdir src/views
 curl https://raw.githubusercontent.com/SAP/luigi/master/core/examples/luigi-example-react/src/views/home.js > src/views/home.js
```
Comment (ndricimrr) [inline scripts/setup/react.sh:54]: Same thing I guess happens here. This curled file uses **"@kyma-project/luigi-client"** as an import. I suppose this is also sth that was fixed in a parallel PR #1311

Comment (ndricimrr) [reply scripts/setup/react.sh:54]: The scripts work well. Leaving this comment here for confused future reviewers. 👍

File: scripts/setup/vue.sh [hunk 3]
```diff
@@ -14,17 +14,87 @@ else
...
-sed 's/luigi-config.js/assets\/luigi-config.js/g' public/index.html > public/index.tmp.html && mv public/index.tmp.html public/index.html
-curl https://raw.githubusercontent.com/SAP/luigi/master/test/e2e-test-application/src/luigi-config/basic/basicConfiguration.js > public/assets/luigi-config.js
-curl https://raw.githubusercontent.com/SAP/luigi/master/test/e2e-test-application/src/assets/sampleexternal.html > public/assets/basicexternal.html
-echo "const webpack=require('webpack');const CopyWebpackPlugin=require('copy-webpack-plugin');module.exports={pages:{sampleapp:{entry:'src/main.js',template:'public/vue.html',filename:'vue.html'}},lintOnSave:true,runtimeCompiler:true,outputDir:'dist',configureWebpack:{plugins:[new CopyWebpackPlugin([{context:'public',to:'index.html',from:'index.html'},{context:'node_modules/@luigi-project/core',to:'./luigi-core',from:{glob:'**',dot:true}},{context:'node_modules/@luigi-project/client',to:'./luigi-client',from:{glob:'**',dot:true}}],{ignore:['.gitkeep','**/.DS_Store','**/Thumbs.db'],debug:'warning'})]}};" > vue.config.js
```
Comment (JohannesDoberer) [inline scripts/setup/vue.sh:29]: Bildschirmfoto 2020-05-18 um 14 32 17
Vue does not work anymore for me

File: scripts/setup/react.sh [hunk 4]
```diff
@@ -15,35 +15,46 @@ npx create-react-app $folder && cd $folder
...
 curl https://raw.githubusercontent.com/SAP/luigi/master/core/examples/luigi-example-react/src/views/sample1.js > src/views/sample1.js
 curl https://raw.githubusercontent.com/SAP/luigi/master/core/examples/luigi-example-react/src/views/sample2.js > src/views/sample2.js
 
 npm i
+npm run buildConfig
 npm start
```
Comment (JohannesDoberer) [inline scripts/setup/react.sh:60]: Bildschirmfoto 2020-05-18 um 14 41 08

File: scripts/setup/react.sh (hunk 2 above)
Comment (maxmarkus) [reply scripts/setup/react.sh:54]: Correct, it requires that other pr to be merged.

File: scripts/setup/vue.sh (hunk 3 above)
Comment (maxmarkus) [reply scripts/setup/vue.sh:29]: It still works for me, did you serve the `..scripts/setup/assets/index.html` from localhost? .. or at least copy it after the curls, right in the beginning of vue startup manually

File: scripts/setup/react.sh (hunk 4 above)
Comment (maxmarkus) [reply scripts/setup/react.sh:60]: Working again, webpack-copy-plugin was released with new version which broke our code, I set it to a fix release version.

File: scripts/setup/vue.sh (hunk 3 above)
Comment (JohannesDoberer) [reply scripts/setup/vue.sh:29]: yes, I did. 
Now it works.. :). All good :)

File: scripts/setup/react.sh (hunk 4 above)
Comment (JohannesDoberer) [reply scripts/setup/react.sh:60]: 👍

**Pull request merged**